src/Query/Query.ts

```diff
@@ -17,6 +17,7 @@ export type GroupingProperty =
     | 'folder'
     | 'happens'
     | 'heading'
+    | 'fn'
```

Comment:
I would prefer `function` rather than `fn`, to make it less cryptic to non-developers...